tarantool_test.go

```diff
@@ -693,6 +694,514 @@ func TestClient(t *testing.T) {
+		}
+		if resp.SQLInfo.AffectedCount != responses[i].SQLInfo.AffectedCount {
+			t.Errorf("Response SQl info is wrong after Execute\nExpected: %d Got: %d\nQuery number: %d",
+				responses[i].SQLInfo.AffectedCount, resp.SQLInfo.AffectedCount, i)
+		}
+		for j := range resp.MetaData.ColumnsInfo {
+			if resp.MetaData.ColumnsInfo[j].FieldIsAutoincrement !=
+				responses[i].MetaData.ColumnsInfo[j].FieldIsAutoincrement {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldIsAutoincrement,
+					resp.MetaData.ColumnsInfo[j].FieldIsAutoincrement, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldIsNullable !=
+				responses[i].MetaData.ColumnsInfo[j].FieldIsNullable {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldIsNullable,
+					resp.MetaData.ColumnsInfo[j].FieldIsNullable, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldCollation !=
+				responses[i].MetaData.ColumnsInfo[j].FieldCollation {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldCollation,
+					resp.MetaData.ColumnsInfo[j].FieldCollation, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldName !=
+				responses[i].MetaData.ColumnsInfo[j].FieldName {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldName,
+					resp.MetaData.ColumnsInfo[j].FieldName, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldSpan !=
+				responses[i].MetaData.ColumnsInfo[j].FieldSpan {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldSpan,
+					resp.MetaData.ColumnsInfo[j].FieldSpan, i)
+			}
+			if resp.MetaData.ColumnsInfo[j].FieldType !=
+				responses[i].MetaData.ColumnsInfo[j].FieldType {
+				t.Errorf("Response MetaData is wrong after Execute\nExpected: %v Got: %v\nQuery number: %d",
+					responses[i].MetaData.ColumnsInfo[j].FieldType,
+					resp.MetaData.ColumnsInfo[j].FieldType, i)
+			}
+		}
+	}
+}
+
+func TestSQLBindings(t *testing.T) {
+	// Data for test table
+	testData := map[int]string{
+		1: "test",
+	}
+
+	// Check for skip SQL tests if tarantool version < 2.0.0
+	isLess, err := test_helpers.IsTarantoolVersionLess(2, 0, 0)
+	if err != nil {
+		t.Errorf("Could not check the Tarantool version")
+		return
+	}
+	if isLess {
+		return
+	}
+
+	var resp *Response
+	var conn *Connection
+
+	conn, err = Connect(server, opts)
+	if err != nil {
+		t.Errorf("Failed to connect: %s", err.Error())
+		return
+	}
+	if conn == nil {
+		t.Errorf("conn is nil after Connect")
+		return
+	}
+	defer conn.Close()
+
+	resp, err = conn.Execute(createTableQuery, []interface{}{})
+	if err != nil {
+		t.Errorf("Failed to Execute: %s", err.Error())
```

Comment:
Errorf do not finishes test case execution, see https://github.com/tarantool/go-tarantool/issues/150 . I'm not sure that this is the expected behavior here.

Reply:
Fixed.